python_scripts/shellies_discovery.py

```diff
@@ -1401,6 +1409,22 @@
     KEY_NAME: "Firmware",
     KEY_STATE_TOPIC: TOPIC_INFO,
 }
+OPTIONS_SENSOR_WINDOW_OPEN = {
```

Comment:
I understand that the user has a window sensor that reports the state to TRV, what is the profit of duplicating this sensor here?

Reply:
Not sure if I understand. Its the state returned from the TRV itself. For example check if TRV has correct window open state assumed. Its always possible that updating TRV window open state gets desynchronized, for example info about window being closed got lost etc. 

TRV get window state update via rest api, and then closes when window state reporting is on and window state got open status. Then you need to report window closed status, some time later.

Reply:
Ok, I understand your point. But I think the name of the sensor is not good. Maybe `Reported window state`?